EventListener/DncGetSubscriber.php

```diff
@@ -0,0 +1,183 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: nbush
+ * Date: 3/5/19
+ * Time: 10:39 AM.
+ */
+
+namespace MauticPlugin\MauticDoNotContactExtrasBundle\EventListener;
+
+use Mautic\CoreBundle\EventListener\CommonSubscriber;
+use Mautic\LeadBundle\Entity\DoNotContact;
+use Mautic\LeadBundle\Entity\Lead;
+use Mautic\LeadBundle\Event\LeadDNCGetEntitiesEvent;
+use Mautic\LeadBundle\LeadEvents;
+
+//use Mautic\LeadBundle\Event\LeadDNCGetCountEvent;
+//use Mautic\LeadBundle\Event\LeadDNCGetEntitiesEvent;
+//use Mautic\LeadBundle\Event\LeadDNCGetListEvent;
```

Comment:
probably can delete this (code cleanup)

Reply:
Working on a bit of clean up already, will push that with the next commit...this is for  out-the-door-now